WalletWasabi/Models/SmartTransaction.cs

```diff
@@ -28,7 +28,8 @@ public class SmartTransaction : IEquatable<SmartTransaction>, IEquatable<Transac
 		public int BlockIndex { get; private set; }
 
 		[JsonProperty]
-		public string Label { get; set; }
+		[JsonConverter(typeof(LabelJsonConverter))]
```

Comment:
```suggestion
		[JsonConverter(typeof(SmartLabelJsonConverter))]
```

Reply:
Thanks, I did it manually instead of accepting your commit suggestions, to make sure I won't miss out anything.